src/components/tabs/tabbed_content/tabbed_content.tsx

```diff
@@ -84,14 +84,19 @@ export type OuiTabbedContentProps = CommonProps &
      * The name property (a node) is also required to display to the user.
      */
     tabs: OuiTabbedContentTab[];
+    /**
+     * Use this prop to control if tab content of unselected tab will stay in dom.
+     */
+    cacheContent?: boolean;
```

Comment:
This isn't really caching...

Some options:
- preserveTabContent
- unmountInactiveTabs
- destroyInactiveTabs
- keepTabsMounted